### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Vasyl-Poremchuk**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,23 +1,42 @@
...
-    pass
+    return LoyaltyProgram.objects.values_list(
+        "name", "bonus_percentage",
+    )
 
 
 def not_active_customers() -> QuerySet:
-    pass
+    return LoyaltyProgramParticipant.objects.filter(
+        last_activity__gt="2021-01-01",
```
Comment (mltdll) [inline app/main.py:18]: You can use `last_activity__year` here.

**Review by mltdll — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (Vasyl-Poremchuk) [reply app/main.py]: Thanks)

**Review by mltdll — APPROVED** [review]
By the way, shorter commit messages are usually better.

**Review by mltdll — APPROVED** [review]
Oops, didn't click "approve".

UPD: Oops, I actually did.

**Review by Nattalli — APPROVED** [review]
Well done!

**Pull request closed without merging**